README.md

```diff
@@ -20,12 +20,14 @@ An image for exporting mysql databases using `gcloud sql export`, and pushing th
 | Variable Name                 | Description                                                         | Default |
 |-------------------------------|---------------------------------------------------------------------|---------|
 | TRACE                         | Enable script tracing                                               | ""      |
+| GCLOUD_VERBOSITY              | Verbosity option passed through to `gcloud`                         | debug   |
+| GCLOUD_WAIT_TIMEOUT           | Timeout (s) passed through to `gcloud sql operations wait`          | 300     |
```

Comment:
is 300s timeout a bit short ?

Reply:
300s is the default.

Generally speaking we rarely see this issue at all, so I would expect the initial request + the additional 300s might be enough.

It's not entirely true that the original request is async - it defiantly blocks...so original command + `wait` might be fine.